CHANGELOG.md

```diff
@@ -5,6 +5,40 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+## [2.1.0] - 2025-09-16
+
+### 🚀 Added
+- **PieceCIDv2 Support**: Added comprehensive support for Piece CID version 2 ([#184](https://github.com/FilOzone/pdp/pull/184))
+  - New `Cids.sol` library with CIDv2 handling capabilities
+  - Support for both CIDv1 and CIDv2 formats 
+  - Enhanced piece data validation using CID height information
+  - Golden tests for CommPv2 functionality
```

Comment:
we ditched version 1 with that change, no backward compat, it's not just "support", it's "switched to"

Reply:
https://github.com/FilOzone/pdp/pull/198/commits/8f73f67cb92e5acf705c5f2bcb71fe0e850b5d8b